specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/InfrastructureInsights.json

```diff
@@ -0,0 +1,336 @@
+        },
+        "Display": {
+            "description": "Contains the localized display information for this particular operation / action.",
+            "properties": {
+                "provider": {
+                    "description": "The localized, friendly version of the resource provider name. The provider name is expected to include the name of the publisher or company responsible. The provider name format should use title case and begin with \"Microsoft\" for first-party services. For example, the provider name may be\"Microsoft Monitoring Insights\" or \"Microsoft Compute\".",
+                    "type": "string"
+                },
+                "resource": {
+                    "description": "The localized, friendly version of the resource type related to this action or operation; the resource type should match the public documentation for the resource provider.",
+                    "type": "string"
+                },
+                "operation": {
+                    "description": "The localized, friendly name for the operation. Use the name as it will be displayed to the user.",
+                    "type": "string"
+                },
+                "description": {
+                    "description": "The localized, friendly description for the operation. The description will be displayed to the user. It should be thorough and concise for used in both tooltips and detailed views.",
+                    "type": "string"
+                }
+            }
+        },
+        "OperationList": {
+            "description": "List of Operations",
+            "properties": {
+                "value": {
+                    "description": "Array of operations",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Operation"
+                    }
+                },
+                "nextLink": {
+                    "description": "URI to the next page of operations.",
+                    "type": "string"
+                }
+            }
+        }
+    },
+    "parameters": {
+        "SubscriptionIdParameter": {
+            "name": "subscriptionId",
+            "in": "path",
+            "description": "Subscription credentials that uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call.",
+            "required": true,
+            "type": "string"
+        },
+        "ApiVersionParameter": {
+            "name": "api-version",
+            "in": "query",
+            "description": "Client API Version.",
+            "required": true,
+            "type": "string",
+            "default": "2016-05-01"
+        },
+        "FilterParameter": {
+            "description": "OData filter parameter.",
+            "name": "$filter",
+            "in": "query",
+            "type": "string",
+            "x-ms-parameter-location": "method"
+        },
+        "RegionParameter": {
```

Comment:
>RegionParameter [](start = 9, length = 15)

A more accurate name could be `LocationParameter`

Reply:
Done.